src/ServiceControl/EventLog/Definitions/ExternalIntegrationEventFailedToBePublishedDefinition.cs

```diff
@@ -1,13 +0,0 @@
-namespace ServiceControl.EventLog.Definitions
-{
-    using ServiceControl.ExternalIntegrations;
-
-    class ExternalIntegrationEventFailedToBePublishedDefinition : EventLogMappingDefinition<ExternalIntegrationEventFailedToBePublished>
```

Comment:
This has just been replaced with a log message right? Which means we don't send it to Pulse and customers won't necessarily know it has happened.